src/volunteers.html

```diff
@@ -1,78 +1,67 @@
-    </div>
-    <div class="col-2">
-      <a href="http://echesters.github.io" class="volunteer-link">
-        <img class="volunteer-picture" src="img/volunteers/Elizabeth_Chesters.png" alt="Elizabeth Chesters Picture">
-        <h2>Elizabeth Chesters</h2>
-      </a>
-      <p class="volunteer-quote">"I joined WHFNP to help me grow in a new career path into user experience, and to expand my portfolio. I also love having side projects with a full team, which gives me motivation and no excuses to procrastinate! The projects have amazing meanings behind them, each with their unique challenges."</p>
-    </div>
-    <div class="col-2">
-      <a href="http://annaleach.net/" class="volunteer-link">
-        <img class="volunteer-picture" src="img/volunteers/Anna_Leach.png" alt="Anna Leach Picture">
-        <h2>Anna Leach</h2>
-      </a>
-      <p class="volunteer-quote">"I joined WHFNP because I was interested in all of the things they do: women in tech, hacking, skills improvement and helping non-profits. I loved it from the first meeting. I always knew taking on a project is one of the best ways to improve your skills and it’s much more fun to do it with lots of other interesting people and a valuable cause" </p>
-    </div>
-    <div class="col-2">
-      <a href="https://uk.linkedin.com/in/raquelalegre" class="volunteer-link">
-        <img class="volunteer-picture" src="img/volunteers/Raquel_Alegre.png" alt="Raquel Alegre Picture">
-        <h2>Raquel Alegre</h2>
-      </a>
-      <p class="volunteer-quote">"Ever since I joined WHFNP in their very first meetup I’ve had the opportunity to practise technologies on real life projects, teach and learn new things from other women in the group, all while making new friends and contributing to good causes. It’s being a great experience!"</p>
-    </div>
-  </div>
-</section>
-<section>
-  <h2 class="section-heading">Do you have any questions?</h2>
-  <p> If you would like to know more check out our <a href="faq.html" class="text-link">FAQ</a>. If your question isn’t answered, feel free to <a href="https://twitter.com/WHFNP" class="text-link">tweet us</a> or <a href="contact.html" class="text-link">drop us a line</a>.</p>
-  <a href="faq.html" class="button">FAQ</a>
-</section>
-<section>
-  <h2 class="section-heading">Start your journey here</h2>
-  <p>Excited to get started? We’re excited to hear from you! Please fill in the form below to join the WHFNP community.</p>
-  <a href="volunteer-contact.html" class="button">Volunteer with us</a>
-</section>
+    --- layout: default title: Volunteers | Women Hack for Non-Profits ---
+    <section class="smaller-banner volunteers-image">
+        <div class="container">
+            <h1 class="title-heading title-cta">Volunteers</h1><a class="button" href="volunteer-contact.html">Volunteer with us</a>
```

Comment:
`title-cta` is not a class anywhere else and has no entry in the css files, please remove.